src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -183,27 +154,25 @@ public void parse_multipleRepeatedFields_acceptsLast() {
     @Test
     public void parse_invalidValueFollowedByValidValue_success() {
         // no other valid values specified
-        Index targetIndex = INDEX_FIRST_PERSON;
-        String userInput = targetIndex.getOneBased() + INVALID_PHONE_DESC + PHONE_DESC_BOB;
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder().withPhone(VALID_PHONE_BOB).build();
+        Index targetIndex = INDEX_FIRST_MODULE;
+        String userInput = targetIndex.getOneBased() + INVALID_CREDITS_DESC + CREDITS_DESC_CS3216;
+        EditModuleDescriptor descriptor = new EditModuleDescriptorBuilder().withModularCredits(VALID_CREDITS_CS3216).build();
         EditCommand expectedCommand = new EditCommand(targetIndex, descriptor);
         assertParseSuccess(parser, userInput, expectedCommand);
 
         // other valid values specified
-        userInput = targetIndex.getOneBased() + EMAIL_DESC_BOB + INVALID_PHONE_DESC + ADDRESS_DESC_BOB
-                + PHONE_DESC_BOB;
-        descriptor = new EditPersonDescriptorBuilder().withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB)
-                .withAddress(VALID_ADDRESS_BOB).build();
+        userInput = targetIndex.getOneBased() + INVALID_CREDITS_DESC + CREDITS_DESC_CS3216;
+        descriptor = new EditModuleDescriptorBuilder().withModularCredits(VALID_CREDITS_CS3216).build();
```

Comment:
I think this test assertion is the same as the one a few lines above :grin: 
Since it's supposed to include other valid values, perhaps we can do:
```suggestion
        userInput = targetIndex.getOneBased() + CODE_DESC_CS2103T +  INVALID_CREDITS_DESC + CREDITS_DESC_CS3216;
        descriptor = new EditModuleDescriptorBuilder().withModuleCode(VALID_CODE_CS2103T).withModularCredits(VALID_CREDITS_CS3216).build();
```